mlir/include/mlir/Dialect/GPU/Transforms/Passes.td

```diff
@@ -87,16 +87,15 @@ def GpuModuleToBinaryPass
     3. `binary`, `bin`: produces binaries.
     4. `fatbinary`, `fatbin`: produces fatbinaries.
   }];
-  let options = [
-    Option<"toolkitPath", "toolkit", "std::string", [{""}],
-           "Toolkit path.">,
-    ListOption<"linkFiles", "l", "std::string",
-           "Extra files to link to.">,
-    Option<"cmdOptions", "opts", "std::string", [{""}],
-           "Command line options to pass to the tools.">,
-    Option<"compilationTarget", "format", "std::string", [{"fatbin"}],
-           "The target representation of the compilation process.">
-  ];
+  let options =
+      [Option<"toolkitPath", "toolkit", "std::string", [{""}], "Toolkit path.">,
+       ListOption<"linkFiles", "l", "std::string", "Extra files to link to.">,
```

Comment:
Don't change the format, just add a new option.

Reply:
git-clang-format does that. Will the checks be happy if I undo it? I am afraid not since the diff is now on the radar. Shall I try?

Reply:
I'm surprised `git clang-format` actually touches .td files!

Reply:
I think it does since the latest version

Reply:
The checks are still happy with the old format but it is probably just a matter of time until it fails there too.